app/questionnaire/navigation.py

```diff
@@ -50,8 +50,20 @@ def build_navigation(self, current_group_id, current_group_instance):
                 else:
                     logger.debug("building navigation", group_id=group['id'])
                     navigation.append(self._build_single_navigation(group, current_group_id, first_location))
+
+        if self._all_sections_completed(navigation):
```

Comment:
How we are originally used 'hide in navigation' was tied to repeating groups, the summary/confirmation made use of this. I think we need to be careful if we start adding everything then hiding somethings